bin/i18n/resources/apps/animations/sync_out.rb

```diff
@@ -34,11 +34,10 @@ def execute
 
               locale = pegasus_lang[:locale_s]
 
-              i18n_spritelab_file_path = CDO.dir(File.join(I18N_LOCALES_DIR, locale, DIR_NAME, SPRITELAB_FILE_NAME))
-              FileUtils.mkdir_p(File.dirname(i18n_spritelab_file_path))
-              FileUtils.mv crowdin_spritelab_file_path, i18n_spritelab_file_path, force: true
-              FileUtils.rm_r File.dirname(crowdin_spritelab_file_path)
-              I18nScriptUtils.delete_empty_crowdin_locale_dir(crowdin_locale)
+              i18n_spritelab_file_path = I18nScriptUtils.locale_dir(locale, DIR_NAME, SPRITELAB_FILE_NAME)
+
+              I18nScriptUtils.rename_dir(File.dirname(crowdin_spritelab_file_path), File.dirname(i18n_spritelab_file_path))
```

Comment:
Do we ever rename without removing? 
If we always rename a file and immediately after we remove it, it may be worth combining `rename_dir` and `remove_dir` into `rename_and_remove_dir`.